department-of-veterans-affairs/va.gov-cms
# Vacms 4856 styling flags in view

## Description

Relates to #6518 .

## Testing done
Filtered for flagged content and saw that flags were pretty.

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/148564889-54def61d-eb55-4f4a-81d8-e3e4de5a8706.png)


## QA steps

As cms admin
1. Go to `/admin/content/flagged` and filter
2. See flags that are styled instead of plain text

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
